sdk/storage/Azure.Storage.Files.Shares/src/Generated/FileRestClient.cs

```diff
@@ -8310,6 +8306,59 @@ internal partial class BrokenLease
 }
 #endregion class BrokenLease
 
+#region class ClearRange
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// ClearRange
+    /// </summary>
+    internal partial class ClearRange
```

Comment:
this type doesn't seem to be different from `FileRange`. should we combine them (also in swagger if applicable) ?

Reply:
This was pattern matched from Page Blobs, and changing it in swagger may affect the other platforms.